### ISSUE TITLE
How can I integrate the_role_bootstrap3_ui in my Rails app?

### ISSUE THREAD

**Issue by egbertp** [TheRole/the_role_management_panel#1]
Hi @the-teacher , thanks for building The Role and the GUI, nice work! 

I'm having some difficulties integrating the GUI to my Rails app. I assume that I have to mount the `TheRoleBoostrap3UI` in my `routes.rb` file. 

I've tried: 

``` rails
mount TheRoleBootstrap3Ui::Engine, at: "/admin"
```

But I still can't access the dashboard via `localhost:3000/admin/roles`. What am I missing? 

Thanks in advance! Best regards,
Egbert

**Comment by the-teacher** [comment]
just add it into gemfile

all routes placed in the_role

TheRoleBootstrap3Ui - it's just set of view files

**Comment by the-teacher** [comment]
do you really need  to mount the_role into your routes.rb?

**Comment by egbertp** [comment]
Let me rephrase: How should I configure my Rails app so that I can use TheRoleBootstrap3Ui? I don't really need to mount them, but at this moment I can't go to `localhost:3000/admin/roles`

**Comment by the-teacher** [comment]
ouch! it's sutuation has not solution right now.
I know how to fix it. Can you waiting for few hours?
I will fix it

**Comment by the-teacher** [comment]
Gem updated for you

https://github.com/the-teacher/the_role#mount-routes

try to install v2.5.4

**Comment by egbertp** [comment]
Thanks for your time and effort! Your update made it possible for me to mount /admin/roles in the application. That's great! 

However, I do not want to user the GUI in combination with my default `layouts/application.html.haml`. Instead I would like the GUI to use its own layout (e.g. `layouts/admin.html.hml`) 

Any suggestions how I can achieve that?

**Comment by the-teacher** [comment]
Please read following section to change layout

https://github.com/the-teacher/the_role#configuration

please close this issue if your problems solved

Good luck! ))

**Comment by the-teacher** [comment]
Hello!

I glad to inform - TheRole 3.0 released.
Few bugs was fixed and code of gem was reworked.

We will be happy to get any feedback from users of TheRole to build new Road Map for gem.

Thank you for your participation!
